src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -70,13 +72,40 @@ case object UnknownPeriod extends Period {
 }
 
 case class PeriodSum(periods: Set[Period], modifier: Modifier) extends Period {
-  override def addTo(temporal: Temporal): Temporal = ???
 
-  override def get(unit: TemporalUnit): Long = ???
+  var map = scala.collection.mutable.Map[TemporalUnit,Long]()
```

Comment:
It's a tiny bit better to construct empty collections like `var map = scala.collection.mutable.Map.empty[TemporalUnit, Long]`.